python/tk_multi_publish2/dialog.py

```diff
@@ -468,7 +468,7 @@ def _on_drop(self, files):
         str_files = []
         for f in files:
             if isinstance(f, unicode):
-                str_files.append(f.encode("utf-8"))
+                str_files.append(f.encode("utf-8").decode("utf-8"))
```

Comment:
this is confusing!

this code now basically takes a unicode string `f` converts it to utf-8 and then converts it back to unicode again:

```
# unicode string
>>> f = u'Manné Öhrström マンネ'
>>> f
u'Mann\xe9 \xd6hrstr\xf6m \u30de\u30f3\u30cd'
>>> print f
Manné Öhrström マンネ

# previous code was converting to utf-8
>>> f.encode("utf-8")
'Mann\xc3\xa9 \xc3\x96hrstr\xc3\xb6m \xe3\x83\x9e\xe3\x83\xb3\xe3\x83\x8d'

# new code converts to utf-8 then converts back
>>> f.encode("utf-8").decode("utf-8")
u'Mann\xe9 \xd6hrstr\xf6m \u30de\u30f3\u30cd'
```

Our general approach in toolkit has to be to convert all unicode to utf-8 at the source or as early as possible - which the previous code already seemed to do. So that every string at runtime in python is a utf-8 encoded stream of bytes. Then we have basically configured QT to treat byte strings as utf-8 when rendering UIs. You can see an example of that code here: https://github.com/shotgunsoftware/tk-maya/blob/master/engine.py#L244

So the bug must be somewhere else - or apologies in advance if i am totally confused! >.<

Maybe it's easier if we do a quick screenshare to go through together?